R/utils.R

```diff
@@ -41,19 +41,6 @@ add_class <- function(x, class) {
   x
 }
 
-`%||%` <- function(lhs, rhs) {
-  lres <- withVisible(eval(lhs, envir = parent.frame()))
-  if (is.null(lres$value)) {
-    eval(rhs, envir = parent.frame())
-  } else {
-    if (lres$visible) {
-      lres$value
-    } else {
-      invisible(lres$value)
-    }
-  }
-}
-
 `%&&%` <- function(lhs, rhs) {
```

Comment:
@krlmlr does this operator exist in other packages? even as utility, I had never seen it.

Reply:
https://github.com/search?q=org%3Acran%20%2F%25%26%26%25%2F&type=code